scripts/ci/clang_ignore.txt

```diff
@@ -0,0 +1,9 @@
+closure/goog/locale/defaultlocalenameconstants.js
```

Comment:
I could've *swore* clang-format-diff.py wasn't respecting the "// clang-format on|off" comments. This was supposed to be a quick workaround, but I just tested it now and it seems to be working. I will remove all of this clang_ignore stuff.